tests/setuponlytests/conflicting_world_for_spigot_server/verify.sh

```diff
@@ -0,0 +1,5 @@
+mc-image-helper assert fileExists world/level.dat && \
+mc-image-helper assert fileExists world_nether/DIM-1/some_spigot_nether_file && \
+mc-image-helper assert fileExists world_the_end/DIM1/some_spigot_end_file && \
+! mc-image-helper assert fileExists world_nether/DIM-1/some_vanilla_nether_file && \
```

Comment:
Nice! I don't even need to add the "not" versions of assertions now that I see this technique.

Reply:
It might still be nice to have "not" versions so that the output can be silent when the test passes and explanatory when it fails, like we get for the non-negated case. Not a huge deal though.

Reply:
Great point. Assertions are easy to add, so I'll get that queued up.